test/src/element/testElementCommands.js

```diff
@@ -96,58 +90,35 @@ describe('element base commands', function() {
     assert.strictEqual(client.reporter.errors, 0);
   });
 
-  it('client.element() W3C Webdriver protocol', async function() {
+  it('client.element() W3C Webdriver protocol', async function () {
     await Nightwatch.initAsync({
       output: false,
       silent: false,
-      selenium: {
-        start_process: false
-      },
       webdriver: {
```

Comment:
- we might be using selenim instead of webdriver here
- to use web driver you can use while initiating Nightwatch client.
```
selenium:{
    host:null
},
```